src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -164,16 +167,119 @@
     colSpan: Int = 1,
     anchor: Int = GridConstraints.ANCHOR_CENTER,
     fill: Int = GridConstraints.FILL_NONE,
-    hSizePolicy: Int = GridConstraints.SIZEPOLICY_FIXED,
-    vSizePolicy: Int = GridConstraints.SIZEPOLICY_FIXED,
+    HSizePolicy: Int = GridConstraints.SIZEPOLICY_FIXED,
```

Comment:
in our own methods, I would like to see compliance with kotlin/java naming guidelines - any reason, you renamed it to have a Capitalized beginning?

Reply:
to easily map (visually) it to the underlying method params. But ok, lets follow camelCase here.